attributes/default.rb

```diff
@@ -530,7 +530,7 @@
 default['hops']['nvidia_pkgs']['download_url']        ="#{node['download_url']}/kube/nvidia"
 
 default['hops']['docker']['git']['image']['name']    = "git"
-default['hops']['docker']['git']['image']['version'] = "0.3.0"
+default['hops']['docker']['git']['image']['version'] = "0.4.0"
```

Comment:
Why do we need to change the version? 
version `0.3.0` is associated with master which is not released yet

Reply:
there is a PR that changes the version